src/System.Private.ServiceModel/src/System/IdentityModel/Tokens/WindowsSecurityToken.cs

```diff
@@ -0,0 +1,156 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.ObjectModel;
+using System.Security.Principal;
+using System.ServiceModel;
+
+namespace System.IdentityModel.Tokens
+{
+
+    public class WindowsSecurityToken : SecurityToken, IDisposable
+    {
+#if FEATURE_NETNATIVE //NegotiateStream
```

Comment:
Hmm, the intent here is that we're stubbing this out while waiting for the S.Net.Security packages to show up. Should we do this even though the intent (hope?) is that we get rid of this in short order? 

If we end up having a substantial implementation difference "for real" then I agree we should split the files.